src-cpp/rdkafkacpp.h

```diff
@@ -187,7 +187,9 @@ enum ErrorCode {
 	/** Produced message timed out*/
 	ERR__MSG_TIMED_OUT = -192,
 	/** Reached the end of the topic+partition queue on
-	 * the broker. Not really an error. */
+	 * the broker. Not really an error. 
+	 * This code is not sent by default since v1.0.0
```

Comment:
This is mentioned in the release notes for v1.0.0, no need to reference the version here.
Suggest rephrasing to: 
```
This event is disabled by default, see the `enable.partition.eof` configuration property.
```
Also update the same error code in src/rdkafka.h accordingly.

Reply:
have adjusted it accordingly. Should be fine now.